### PR TITLE
Update Board - MatekF405 Wing.md

### PR DESCRIPTION
**Pull request by JuliusZimbovsky**
added link to  matek board photo

### CODE REVIEW

**Comment by JulioCesarMatias** [comment]
LGTM! Pawel link is broken.

File: docs/Board - MatekF405 Wing.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Board - [MATEKSYS F405-Wing](https://inavflight.com/shop/p/MATEKF405WING)
 
-![Matek F405 Wing](https://quadmeup.com/wp-content/uploads/2018/12/DSC_0007.jpg)
+![Matek F405 Wing](http://www.mateksys.com/?portfolio=f405-wing)
```
Comment (DzikuVx) [inline docs/Board - MatekF405 Wing.md:3]: Well, this a website link, while we expect image URL over here. Please update to point to image, not website

**Review by DzikuVx — CHANGES_REQUESTED** [review]

**Comment by JulioCesarMatias** [comment]
this image seems to me to be the best:

http://www.mateksys.com/wp-content/uploads/2018/04/F405-WING_Layout-1471x1500.jpg

**Comment by JulioCesarMatias** [comment]
> LGTM! Pawel link is broken.


I didn't realize that the link went to Matek's website, I thought it went to a direct image. Sorry

**Pull request closed without merging**